float32 value
float32 fix_duration
---
